src/actions.ts

```diff
@@ -31,12 +31,41 @@ export function getDependencies(
   return { keys, selected, selections };
 }
 
-export async function installTypes(dependencies: string[], { toDev = false, selections, pwd = '', concurrency = 1 }) {
-  const { stdout: yarnPath } = await shell('which yarn');
+function getYarnVersion() {
+  return shell('yarn --version')
+    .then((result) => result.stdout)
+    .catch(() => '');
+}
 
-  const installer = !yarnPath ? 'npm install --save' : `yarn add`;
-  const directory = pwd ? `cd ${pwd} &&` : '';
+export interface InstallTypesOptions {
```

Comment:
Prefix interfaces with I. This should be picked up by lint... Lint may need updating later if not.